flixel/system/FlxAssets.hx

```diff
@@ -104,7 +104,6 @@ class FlxAssets
 	 *                          will exclude .ogg files and everything in the exclude folder
 	 * @param   rename          A function that takes the file path and returns a valid haxe field name.
 	 *
-	 * @see [AssetPaths](https://api.haxeflixel.com/AssetPaths.html) for more examples
```

Comment:
I think `AssetPaths` can be added to api.flixel.com, it has now good doc and examples, thus the link should be here.

Reply:
figure out how to add it, first. api.flixel pages are generated from from files in [this directory](https://github.com/HaxeFlixel/flixel/tree/dev/flixel), as I said before, AssetPaths doesn't exists in this repo, its created when the dev creates a project.

I also don't think we should add it to the api, because it's not actually part of the api. Different games can have different implementations on AssetPath. this is why we added docs to the template

Reply:
>it's not actually part of the api.

`spinehaxe` and `nape` too, yet [they have their doc](https://api.haxeflixel.com/spinehaxe/). They are explicitly added to the doc generation process in [project.xml](https://github.com/HaxeFlixel/flixel-docs/blob/c2a3205a94558d9d39eee831727b34e57f3794c8/api/xml/project.xml#L13-L16).

>figure out how to add it

There are two options:

1. The shortest: update docgen script to download AssetPaths.hx and save it next to [`project.xml`](https://github.com/HaxeFlixel/flixel-docs/blob/master/api/xml/project.xml). Include AssetPaths.hx as haxeflag. I did it, and it worked:
![image](https://user-images.githubusercontent.com/9349164/223090798-418464cd-7cde-4aca-a6a4-846b34df2cff.png)
2. Longer, but probably better: alter `flixel-templates` library so it provides AssetPaths as a real haxe module, then setup docgen `project.xml` to use it.